test/require-config.js

```diff
@@ -6,30 +6,31 @@ require.config({
     baseUrl: '/base',
     waitSeconds: 60,
     paths: {
-        backbone: 'node_modules/backbone/backbone',
+        'backbone': 'node_modules/backbone/backbone',
```

Comment:
Sure. The rule is [`quote-props`](http://eslint.org/docs/rules/quote-props), if you're interested in opening up a discussion about it, maybe file an issue to the eslint-config-edx repo and tag people?

(I have Opinions about why `consistent-as-needed` is the right way to do object property quoting but I think they should be aired over on that repo, not in this PR)